list-and-titles-of-products.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this PR is to discuss and approve the list and correct titles of all products that we will indicate on the website in the near future.
+
+### Financial area
+
+Ubiquity Dollar 
+
+Ubiquity Card
```

Comment:
UbiquiCard is cute but maybe not what we should really call it. I suppose Ubiquity Card makes the most sense. 

Other ideas:
- Ubiquity Credit
- Ubiquity Debit